test/kubemark/gce/util.sh

```diff
@@ -45,8 +45,8 @@ function create-kubemark-master {
     KUBE_GCE_INSTANCE_PREFIX="${KUBE_GCE_INSTANCE_PREFIX:-e2e-test-${USER}}-kubemark"
     SCALEOUT_PROXY_NAME="${KUBE_GCE_INSTANCE_PREFIX}-proxy"
     if [[ "${KUBERNETES_RESOURCE_PARTITION:-false}" == "true" ]]; then
-      KUBECONFIG="${RP_KUBECONFIG}"
-      KUBE_GCE_INSTANCE_PREFIX="${KUBE_GCE_INSTANCE_PREFIX}-rp"
+      KUBECONFIG="${RP_KUBECONFIG}-${RESOURCE_PARTITION_SEQUENCE}"
```

Comment:
FYI. as we discussed last week. the kubeconfig variable will be set in start-kubemark.sh and subprocess will use the one set. this could cause conflict when merging code.